gallery/src/components/NxModal/NxModalEscExample.tsx

```diff
@@ -44,9 +44,9 @@ export default function NxModalStackedExample() {
         </div>
       }
       {showModal &&
-        <NxModal id="nx-modal-esc-example-modal" onCancel={modal1CancelHandler}>
+        <NxModal id="nx-modal-esc-example-modal" onCancel={modal1CancelHandler} aria-labelledby="modal-esc-header">
           <header className="nx-modal-header">
-            <h2 className="nx-h2">
+            <h2 className="nx-h2" id="modal-esc-header">
```

Comment:
MIssing label in the other modal

Reply:
> MIssing label in the other modal

Added: https://github.com/sonatype/sonatype-react-shared-components/pull/440/commits/90f65c1b76c9c3e995743f184c788f721810fefa